packages/clay-alert/src/timerHook.tsx

```diff
@@ -0,0 +1,39 @@
+/**
+ * © 2019 Liferay, Inc. <https://liferay.com>
+ *
+ * SPDX-License-Identifier: BSD-3-Clause
+ */
+
+import * as React from 'react';
+
+export default (autoClose?: boolean | number, onClose = () => {}) => {
```

Comment:
I think we need to improve on the logic here of timer administration, some things in the demo do not work so well.